lib/FinderNavigation/styles.scss

```diff
@@ -32,7 +32,7 @@ $finder-indent-level-support: 10 !default;
 
   .button:hover path,
   .dropdown-gc.is-active path,
-  &.is-active .finder__item-link path {
+  &.is-active .finder__item--link path {
```

Comment:
I thought `.finder__item--link` was an element so `.finder__item-link` fits with that.

It seems to be applied to a link meaning that it is an element and not a modifier of an existing element within the component.

I recommend we retain `finder__item-link` and change our usage in the app to use this class over changing it in here to break BEM naming conventions.

Reply:
Ah, i thought it was the other way round!